source/includes/_scarts.md

```diff
@@ -0,0 +1,15 @@
+# SCA-RTS (PS21/19)
+
+## Overview
+In November 2021, the FCA set out new rules for standards on Strong Customer Authentication (SCA) in a change to the Regulatory Technical Standards on Strong Customer Authentication and Secure Communication (SCA-RTS).
+
+We (Monzo) will release these changes as of 2022-08-01. This will not be a breaking change.
+
+All user authenticated 90-day consents will continue to be valid (unless expired as of 2022-08-01). After this date, when a 90-day consent expires the user will be required to re-authenticate which will create a “long-lived” consent.
```

Comment:
*Only AIS consents. 
* I find the phrasing a bit confusing, how about "Consents created before this date will expire after 90 days like before. Consents created after this date will be long lived"